### ISSUE TITLE
I would like to be able to have a different Layout position for objects that appear in multiple layers. I see layers as a way to define Subject Areas of a data model. And I typically use Subject Areas to focus on a small set of tables for discussion. It would be beneficial to layout these objects on a per payer basis

### ISSUE THREAD

**Issue by bille67** [pgmodeler/pgmodeler#1318]
**Feature description**
_A clear and concise description of what the problem is._

**Sample image**
_If the feature requested is a visual improvement, please, attach some images to make it clear._

**Additional info**
_Add any other context or screenshots about the feature request here._

**Comment by Maxzor** [comment]
Having positions of the graphical items change, according to layers visible, would be quite an endeavour code-wise, but definitely a great feature IMO
An idea in the direction of 2D projection of high-dimension categories...

**Comment by rkhaotix** [comment]
Yes. It's quite complex and resource consuming to have multiple scenes holding the same item in different positions. 

I have some solutions in mind but I will start to work on it only during the development of 0.9.3 because right now I freezing the new feature development due to the release of 0.9.2 in the end of this month.

**Comment by metapsj** [comment]
fwiw, the concept of subject areas would be super helpful.  in tools like erwin and er/studio they have tabs that expose different canvases where a subject area can be defined while maintaining the main canvas' definitions.
